# The Idea

# The Code

In [1]:
expensive_hobbies = {'Mountaineering': 10000, 'Skydiving': 5e3, 'Salsa': 0,
                    'Scuba_diving': 0, 'Rock_climbing': 1e3, 'Plane_tickets': 5e3 }

In [9]:
monthly_hobby_costs = sum(expensive_hobbies.values()) / 12
monthly_budget = 4e3 

In [10]:
monthly_hobbie_costs

1750.0

In [11]:
total_budget = monthly_budget + monthly_hobby_costs

## The Four Percent Rule

In [16]:
"${:,}".format(total_budget * 12 * 25)

'$1,725,000.0'

In [19]:
cushion = total_budget * 12 * 4

In [22]:
"${:,}".format(cushion)

'$276,000.0'

In [25]:
(total_budget * 12 - (cushion * 0.08) ) / 12

3910.0

So we just need to make 3910 a month and then we are good to go. This is Gili T profits for sure but there is more. 

# A Middle Ground

It takes a long time to save up 1.7MM so this answer us rather dissatisfying. A better idea may rather be to have a large capital cushion of 5 years living expenses and finding sources of passive/alternative income for the rest of the money. 

# My Ways to do it

1. Real Estate
2. Multi-Unit real estate
3. Make Data Science consultancy
4. [List of Side hustles](https://www.evernote.com/shard/s11/nl/324799287/fc3d4207-1980-42ff-97e9-dba808be00d8/)
    - The language school idea in particular I like quite a lot.